### ISSUE TITLE
GitHub CI is broken

### ISSUE THREAD

**Issue by skyzh** [bytedance/terarkdb#83]
### Problem

* Currently, GitHub Action is not running for new PRs. https://github.com/bytedance/terarkdb/pull/84#event-4964716904
* <del>There is no `make test` target in generated Makefile if using commands in the workflow.</del> Fixed, but unit test will fail. https://github.com/bytedance/terarkdb/issues/85
* `clang-format` is not checked in CI scripts.

### Solution

I'm looking into fixes for this repo.

**Comment by jackwener** [comment]
I am willing to take this task up.

**Comment by skyzh** [comment]
There is only one test still failing `db_compaction_test`. You are welcomed to have a try!

**Comment by jackwener** [comment]
Maybe I can add format-check in CI first.

**Comment by jackwener** [comment]
> Maybe I can add format-check in CI first.

Based on the dev.1.4 branch, there are many important branches under development, so it was decided to suspend clang-format

**Comment by skyzh** [comment]
Yes. We have produced a 10000+ LoCs of diff with clang-format. This may make current developing features hard to merge into master. We may format them as merging, and the codebase will pass clang-format check months later.
